src/sql/src/plan/statement/ddl.rs

```diff
@@ -1163,6 +1163,34 @@ pub fn plan_create_view(
     }))
 }
 
+pub fn describe_create_views(
+    _: &StatementContext,
+    _: CreateViewsStatement<Raw>,
+) -> Result<StatementDesc, anyhow::Error> {
+    Ok(StatementDesc::new(None))
+}
+
+pub fn plan_create_views(
+    scx: &StatementContext,
+    stmt: CreateViewsStatement<Raw>,
+) -> Result<Plan, anyhow::Error> {
+    match stmt.views_definition {
+        CreateViewsDefinition::Literal(views) => {
+            let mut planned_views = Vec::with_capacity(views.len());
+            for view in views {
+                match plan_create_view(scx, view, &Params::empty())? {
+                    Plan::CreateView(plan) => planned_views.push(plan),
+                    _ => unreachable!(),
+                }
+            }
+            Ok(Plan::CreateViews(CreateViewsPlan {
+                views: planned_views,
+            }))
+        }
+        CreateViewsDefinition::Source { .. } => bail!("cannot plan unpurified statement"),
```

Comment:
`cannot create view from source` might be a more meaningful error statement for the user!

Reply:
good point, I'll fix this at merge time to not rebase everything just yet